src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Scalar.cs

```diff
@@ -334,6 +334,46 @@ public static T Convert(int value)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static T CopySign(T value, T sign)
+        {
+            // byte, ushort, uint, and ulong should have already been handled
```

Comment:
Would have already been handled where? If vectorization isn't supported and we hit this fallback path, we couldn't end up here directly from a call to the public `VectorXx<T>.CopySign`?

Reply:
The vector APIs all have a fast path to handle `no-op` operations and avoid the for loop/call to `Scalar<T>.CopySign` entirely

This also benefits other runtimes (like Mono), which don't always handle generics/inlining as well today.